Frameworks/UIKit/UIImage.mm

```diff
@@ -123,6 +122,11 @@ + (UIImageCachedObject*)cacheImage:(UIImage*)image withName:(NSString*)name {
     return [obj autorelease];
 }
 
+static inline CGImageRef getImage(UIImage* self) {
+    RETURN_NULL_IF(!self);
```

Comment:
> RETURN_NULL_IF(!self) [](start = 3, length = 22)

This is actually an overkill and unnecessary.  If self is null, we should crash.  You can send message to nil object, but can't access ivars of nil objects.  failing fast by crashing will help catch issues sooner.